.github/CODEOWNERS

```diff
@@ -134,7 +134,7 @@
 /pkgs/development/ruby-modules      @marsam
 
 # Rust
-/pkgs/development/compilers/rust @Mic92 @LnL7 @zowoq @winterqt @figsoda
+/pkgs/development/compilers/rust @Mic92 @zowoq @winterqt @figsoda
```

Comment:
I'm surprised that @LnL7 doesn't have write access. Should we grant that instead of removing here?

Reply:
That commit is already on master after a rebase, so this not really up for discussion here.